app/victoria-metrics/main_test.go

```diff
@@ -238,16 +236,27 @@ func testWrite(t *testing.T) {
 func testRead(t *testing.T) {
 	for _, engine := range []string{"prometheus", "graphite", "opentsdb", "influxdb", "opentsdbhttp"} {
 		t.Run(engine, func(t *testing.T) {
-			for _, x := range readIn(engine, t, fmt.Sprintf("%d", insertionTime.UnixNano())) {
+			for _, x := range readIn(engine, t, insertionTime) {
 				test := x
 				t.Run(test.Name, func(t *testing.T) {
 					t.Parallel()
 					for _, q := range test.Query {
+						q = testutil.PopulateTimeTplString(q, insertionTime)
+						t.Log("QEURY", q)
+						log.Println("QEURY", q)
 						switch true {
 						case strings.HasPrefix(q, "/api/v1/export"):
 							checkMetricsResult(t, httpReadMetrics(t, testReadHTTPPath, q), test.ResultMetrics, q, test.Issue)
 						case strings.HasPrefix(q, "/api/v1/series"):
-							checkSeriesResult(t, httpReadSeries(t, testReadHTTPPath, q), test.ResultSeries, q, test.Issue)
+							s := Series{}
+							httpReadStruct(t, testReadHTTPPath, q, &s)
+							checkSeriesResult(t, s, test.ResultSeries, q, test.Issue)
+						case strings.HasPrefix(q, "/api/v1/query_range"):
```

Comment:
how this is different to `default` case?

Reply:
query contains query_range suffix. just want avoid mistake in future